File: hackertool/every_ip_address.py
#!/usr/bin/env pypy

# Print every IPv4 address in the same format as the torrent
# First try used python3 before I realized pypy would be way faster before I
# realized C/C++ would be fastest and also really easy to write
# mal <mal@sec.gd>

for a in range(0,256):
    for b in range(0,256):
        for c in range(0,256):
            for d in range(0,256):
                print str(a) + "." + str(b) + "." + str(c) + "." + str(d)

File: hackertool/every_ip_address.cpp
// Print every IPv4 address in the same way as the torrent
// mal <mal@sec.gd>

#include <iostream>

int main(){
    for(int a=0;a<=255;a++){
        for(int b=0;b<=255;b++){
            for(int c=0;c<=255;c++){
                for(int d=0;d<=255;d++){
                    std::cout << a << '.' << b << '.' << c << '.' << d << '\n';
                }
            }
        }
    }
}
